core/src/test/java/org/apache/accumulo/core/security/ColumnVisibilityTest.java

```diff
@@ -19,22 +19,26 @@
 package org.apache.accumulo.core.security;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
-import static org.apache.accumulo.core.security.ColumnVisibility.quote;
 import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import java.util.Comparator;
-
-import org.apache.accumulo.core.security.ColumnVisibility.Node;
-import org.apache.accumulo.core.security.ColumnVisibility.NodeComparator;
-import org.apache.accumulo.core.security.ColumnVisibility.NodeType;
 import org.apache.hadoop.io.Text;
 import org.junit.jupiter.api.Test;
 
 public class ColumnVisibilityTest {
 
+  @SuppressWarnings("deprecation")
```

Comment:
Seeing these changes made me realize that maybe we could start removing some of the stuff that was deprecated re ColumnVisibility in 4.0.0.